oap-server/server-starter/src/main/resources/ui-initialized-templates/flink/flink-jobManager.json

```diff
@@ -0,0 +1,639 @@
+                    "tips": "The amount of the jobManager JVM nonHeap memory used"
+                  }
+                },
+                {
+                  "x": 0,
+                  "y": 8,
+                  "w": 3,
+                  "h": 8,
+                  "i": "8",
+                  "type": "Widget",
+                  "id": "0-0-8",
+                  "moved": false,
+                  "expressions": [
+                    "latest(meter_flink_jobManager_taskManagers_slots_total)"
+                  ],
+                  "typesOfMQE": [
+                    "SINGLE_VALUE"
+                  ],
+                  "graph": {
+                    "type": "Card",
+                    "fontSize": 20,
+                    "textAlign": "center",
+                    "showUnit": true
+                  },
+                  "widget": {
+                    "name": "Task Managers Slots Total",
+                    "title": "Total Slots",
+                    "tips": "The number of total slots"
+                  }
+                },
+                {
+                  "x": 3,
+                  "y": 0,
+                  "w": 3,
+                  "h": 8,
+                  "i": "9",
+                  "type": "Widget",
+                  "id": "0-0-9",
+                  "moved": false,
+                  "expressions": [
+                    "latest(meter_flink_jobManager_taskManagers_slots_available)"
+                  ],
+                  "typesOfMQE": [
+                    "SINGLE_VALUE"
+                  ],
+                  "graph": {
+                    "type": "Card",
+                    "fontSize": 20,
+                    "textAlign": "center",
+                    "showUnit": true
+                  },
+                  "widget": {
+                    "name": "Task Managers Slots Available",
+                    "tips": "Task Managers Slots Available",
+                    "title": "The number of available slots"
+                  }
+                },
+                {
+                  "x": 15,
+                  "y": 0,
+                  "w": 9,
+                  "h": 16,
+                  "i": "13",
+                  "type": "Widget",
+                  "id": "0-0-13",
+                  "moved": false,
+                  "expressions": [
+                    "meter_flink_jobManager_jvm_cpu_time/1000/1000"
+                  ],
+                  "typesOfMQE": [
+                    "TIME_SERIES_VALUES"
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "title": "JVM CPU Time(ms)",
+                    "name": "JVM CPU Time",
+                    "tips": "The jobManager CPU time used by the JVM"
```

Comment:
MAL `increase('PT1M')`,  means the number increased in 1 min. Please check other places.